Entities/Entities/Model1.Context.cs

```diff
@@ -0,0 +1,35 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     Этот код создан по шаблону.
+//
+//     Изменения, вносимые в этот файл вручную, могут привести к непредвиденной работе приложения.
+//     Изменения, вносимые в этот файл вручную, будут перезаписаны при повторном создании кода.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Entities
+{
+    using System;
+    using System.Data.Entity;
+    using System.Data.Entity.Infrastructure;
+    
+    public partial class EntitiesDBEntities : DbContext
+    {
+        public EntitiesDBEntities()
+            : base("name=EntitiesDBEntities")
+        {
+        }
+    
+        protected override void OnModelCreating(DbModelBuilder modelBuilder)
+        {
+            throw new UnintentionalCodeFirstException();
+        }
+    
+        public virtual DbSet<course_lecturers> course_lecturers { get; set; }
+        public virtual DbSet<courses> courses { get; set; }
```

Comment:
since all classes and properties by codestyle should be by uppercase, here some problems

Reply:
you everytime should use this code style, no matter what you see in task or somewhere else

Reply:
As u can see in the top of the file it's auto-generated code, I didn't write anything in this file